src/OAuth/Tokens/PenneoTokens.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Penneo\SDK\OAuth\Tokens;
+
+/** @internal */
+final class PenneoTokens
+{
+    private $refreshToken;
+
+    private $accessToken;
+
+    private $accessTokenExpiresAt;
+
+    private $refreshTokenExpiresAt;
+
+    public function getRefreshToken(): string
+    {
+        return $this->refreshToken;
+    }
+
+    public function getAccessToken(): string
+    {
+        return $this->accessToken;
+    }
+
+    public function getAccessTokenExpiresAt(): int
+    {
+        return $this->accessTokenExpiresAt;
+    }
+
+    public function getRefreshTokenExpiresAt(): int
+    {
+        return $this->refreshTokenExpiresAt;
+    }
+
+    public function __construct(
+        string $accessToken,
+        string $refreshToken,
+        int $accessTokenExpiresAt,
+        int $refreshTokenExpiresAt
+    ) {
+        $this->refreshToken = $refreshToken;
+        $this->accessToken = $accessToken;
+        $this->accessTokenExpiresAt = $accessTokenExpiresAt;
+        $this->refreshTokenExpiresAt = $refreshTokenExpiresAt;
+    }
+
+    public function serialize(): string
```

Comment:
So that we can store and load the tokens.

> Why do we need to store and load the tokens?

To allow for integrations to utilise the SDK across multiple requests. Our silverfin adapter would be a good example for this. After authorisation the tokens are stored in the session and later used when users initiate requests to import documents.

Reply:
Oh, sorry. Thought I deleted this message as the answered gave itself when I came cross the session storage 😅